raiden-dapp/tests/unit/app.spec.ts

```diff
@@ -6,29 +6,30 @@ import Vue from 'vue';
 import VueRouter from 'vue-router';
 import Vuex from 'vuex';
 import Vuetify from 'vuetify';
-import Mocked = jest.Mocked;
 import store from '@/store/index';
 import RaidenService from '@/services/raiden-service';
 import App from '@/App.vue';
+import Mocked = jest.Mocked;
 
 Vue.use(VueRouter);
 Vue.use(Vuex);
 Vue.use(Vuetify);
 
 describe('App.vue', () => {
-  let wrapper: Wrapper<App>;
   let vuetify: Vuetify;
   let router: Mocked<VueRouter>;
   let $raiden: RaidenService;
 
-  beforeEach(() => {
+  const createWrapper = (enableImprint = false): Wrapper<App> => {
```

Comment:
I think according to our test style guide the wrapper function should be defined outside of the `describe` block.

Reply:
Done!